cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -229,10 +229,56 @@ pScriptFor name (Just deprecated) help =
         <> Opt.internal
         )
 
+pPlutusReferenceScriptWitnessFiles
+  :: WitCtx witctx
+  -> BalanceTxExecUnits -- ^ Use the @execution-units@ flag.
+  -> Parser (ScriptWitnessFiles witctx)
+pPlutusReferenceScriptWitnessFiles witctx autoBalanceExecUnits =
+   toReferenceScriptWitnessFiles
+     <$> ( (,,,,) <$> pReferenceTxIn
+                  <*> pAnyScriptLang
+                  <*> pScriptDatumOrFile "reference-tx-in" witctx
+                  <*> pScriptRedeemerOrFile "reference-tx-in"
+                  <*> (case autoBalanceExecUnits of
+                        AutoBalance -> pure (ExecutionUnits 0 0)
+                        ManualBalance -> pExecutionUnits "reference-tx-in")
+         )
+ where
+  pReferenceTxIn :: Parser TxIn
+  pReferenceTxIn =
+    Opt.option (readerFromParsecParser parseTxIn)
+      (  Opt.long "tx-in-reference"
+      <> Opt.metavar "TX-IN"
+      <> Opt.help "TxId#TxIx - Specify a reference input. The reference input may or may not have\
+                  \ a plutus reference script attached."
+      )
+  pAnyScriptLang :: Parser AnyScriptLanguage
+  pAnyScriptLang =
+    Opt.flag' (AnyScriptLanguage $ SimpleScriptLanguage SimpleScriptV1)
+      (  Opt.long "simple-script-v1"
+      <> Opt.help "Specify a simple script v1 reference script. \
+                  \See documentation at doc/reference/simple-scripts.md"
```

Comment:
This document doesn't refer to v1 and v2. How can we make that distinction clear?

Reply:
`Specify a simple script v1 (Shelley) reference script`
`Specify a simple script v2 (Allegra) reference script.`
?